File: alpro-lioneljevon-16-10-2025/konversi-suhu.py
print("Menghitung Konversi Suhu Celsius ke Fahrenheit dan Celsius ke Reamur \n")

celsiusFahrenheit = lambda celsius : (9/5) * celsius + 32
celsiusReamur = lambda celsius : (4/5) * celsius

celsius = float(input("Masukkan Suhu dalam Celsius: "))

print("Hasil Konversi Celsius ke Fahrenheit adalah:", celsiusFahrenheit(celsius))
print("Hasil Konversi Celsius ke Reamur adalah:", celsiusReamur(celsius))

File: alpro-lioneljevon-16-10-2025/Versi C++/konversi-suhu.cpp
#include <iostream>
using namespace std;

double celsiusFahrenheit(double celsius) {
    return (9.0 / 5.0) * celsius + 32;
}

double celsiusReamur(double celsius) {
    return (4.0 / 5.0) * celsius;
}


int main() {
    double celsius;
    cout << "Masukkan Suhu Celsius : "; cin >> celsius;
    cout << celsius << endl;

    double fahrenheit = celsiusFahrenheit(celsius);
    double reamur = celsiusReamur(celsius);
    cout << "Hasil Celsius ke Fahrenheit : " << fahrenheit << endl;
    cout << "Hasil Celsius ke Reamur : " << reamur << endl;

    return 0;
}